src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_BALL = 101;
+    private ColorSupplier color = new ColorSupplier();
+    private Random random = new Random();
+
+    public String getRandomBall() {
+        int num = random.nextInt(MAX_NUMBER_BALL);
+        String col = color.getRandomColor();
+        return num + " " + col;
+    }
```

Comment:
here, you should create a Ball object
Ball class needs toString method overrided - which will be used to get String representation of Ball object, instead of constructing that string here
Also, be mindful of informative naming - you don't save much time not typing out a few extra letters, but that makes code harder to read for others. Remember, that we spend more times reading others code, than writing our own :)
Change to number, color, use a ball constructor